test/functional/apps/discover/_discover.js

```diff
@@ -115,7 +115,7 @@ export default function ({ getService, getPageObjects }) {
         await PageObjects.discover.waitUntilSearchingHasFinished();
 
         const newDurationHours = await PageObjects.timePicker.getTimeDurationInHours();
-        expect(Math.round(newDurationHours)).to.be(24);
+        expect(Math.round(newDurationHours)).to.be(23);
```

Comment:
Seems like these tests are skipped anyway - why the change? And what's causing it? Is this a search/replace artifact?

Reply:
time to unskip :) thanks for the info! since the layout changed, the chart component has a different width and but the brushing width hasn't changed, that's why a different time range is selected here